src/warplib/generateWarplib.ts

```diff
@@ -64,3 +67,20 @@ export const warplibFunctions: WarplibFunctionInfo[] = [
 ];
 
 warplibFunctions.forEach((warpFunc: WarplibFunctionInfo) => generateFile(warpFunc));
+
+// generate warplib lib
+fs.writeFileSync(
+  path.join('.', 'warplib', 'lib.cairo'),
+  endent`
+    // Add here warplib modules in cairo1
+    mod integer;
+  `,
+);
+
+// generate warplib/maths lib
+fs.writeFileSync(
+  path.join('.', 'warplib', 'maths', 'lib.cairo'),
+  endent`
+    // Add here maths modules in cairo1
+  `,
+);
```

Comment:
Is there any reason to have that generated? I mean won't static files do the job?

Reply:
I am not sure I understand what you mean. 

My first attempt was to make `maths` a sub module of `warplib`. So to import anything from `maths` we would do something like `warplib::maths::func`, but when `maths` is set as a submodule of `warplib` it expects `math` to be a Cairo file and not a folder containing other cairo files

Reply:
I mean why don't we simply commit those files to repo? Why do we need `fs.writeFileSync(`? You write static string to a static path anyway